internal/file/finder.go

```diff
@@ -108,6 +113,38 @@ func (finder *Finder) GetGroups(options DebrickedOptions) (Groups, error) {
 			return nil
 		},
 	)
+	noGroupsFound = len(groups.groups) == 0
```

Comment:
can we break out this logic to a separate function? It is quite large with these additions, also a lot of nested statements